src/pages/CreatePool.tsx

```diff
@@ -175,6 +179,49 @@ export default function CreatePool(): React.ReactElement {
           </Button>
         </Box>
       </Paper>
+      <Dialog open={true}>
```

Comment:
Could you also hook this up with component state. This part isn't dependent on any web3 interactions.